### ISSUE TITLE
How to get dc:description ?

### ISSUE THREAD

**Issue by romaingiard** [readium/SDKLauncher-Android#105]
Hello,

Is-it possible to get dc:description in the EPub, it seems that the tag is not loaded in objet Package (Package.java) .

Thanks for help.

**Comment by rkwright** [comment]
@romaingiard You are correct.  The DC:description is not exposed.  If you would like to implement this, please feel free to do so and create a pull request.  You should take a look at this PR:
https://docs.google.com/document/d/1-CCiMpQsR5tu_HdbmiHoXMJQJ1E6T4RQbWYoDlXGpuM/edit#
which implements that functionality for some of the other Dublin Core metadata.

**Comment by danielweck** [comment]
Also see the recent dc:publisher PR: https://github.com/readium/readium-sdk/pull/258/files
